pages/api/token/temporary.ts

```diff
@@ -0,0 +1,111 @@
+import dbConnector from "@lib/integration/dbConnector";
+import executeQuery from "@lib/integration/queryManager";
+import { logMessage } from "@lib/logger";
+import validate from "@lib/middleware/bearerToken";
+import { BearerTokenPayload } from "@lib/types";
+
+import jwt from "jsonwebtoken";
+import { NextApiRequest, NextApiResponse } from "next";
+import { QueryResult } from "pg";
+
+/**
+ * Verifies that the payload for the request is valid.
+ *
+ * @param handler - the function that will be called from the promise
+ * @returns a promise
+ */
+const checkRequestPayload = (
+  handler: (
+    req: NextApiRequest,
+    res: NextApiResponse,
+    bearerTokenPayload: BearerTokenPayload,
+    email: string
+  ) => void
+) => {
+  return async (req: NextApiRequest, res: NextApiResponse, options?: unknown): Promise<unknown> => {
+    const requestBody = JSON.parse(req.body);
+    if (requestBody?.email) {
+      return handler(req, res, options as BearerTokenPayload, requestBody["email"]);
+    } else {
+      res.status(400).json({ error: "Invalid payload" });
+    }
+  };
+};
+
+/**
+ * Verifies that the payload for the request is valid.
+ *
+ * @param req - the api request
+ * @param res - the api response
+ * @param bearerTokenPayload - the payload from the bearer token used in the authorization
+ * @param email - the email for the user making the request, found in the body of the api request
+ * @returns a promise
+ */
+const handler = async (
+  req: NextApiRequest,
+  res: NextApiResponse,
+  bearerTokenPayload: BearerTokenPayload,
+  email: string
+) => {
+  try {
+    const formID = bearerTokenPayload.formID;
+    if (formID == undefined) {
+      res.status(403).json({ error: "Invalid form request." });
```

Comment:
Really hate to nit here. Think I mentioned it before in a previous review you can just do the return on the same line as the res.status().json(). We do this across our code base so might as well keep it consistent. Also looks cleaner than having an empty return line.

https://github.com/cds-snc/platform-forms-client/pull/557#discussion_r770649977